Sources/OpenAPIRuntime/Base/Base64EncodedData.swift

```diff
@@ -74,7 +80,7 @@ extension Base64EncodedData: Codable {
         guard let data = Data(base64Encoded: base64EncodedString, options: options) else {
             throw RuntimeError.invalidBase64String(base64EncodedString)
         }
-        self.init(data: ArraySlice(data))
+        self.init(ArraySlice(data))
```

Comment:
With the change above, this should work by passing `Data` directly.
```suggestion
        self.init(data)
```